scarb/tests/cache.rs

```diff
@@ -0,0 +1,28 @@
+use assert_fs::prelude::*;
+
+use scarb_test_support::command::Scarb;
+use scarb_test_support::project_builder::ProjectBuilder;
+
+#[test]
+fn simple_clean() {
+    let t = assert_fs::TempDir::new().unwrap();
```

Comment:
```suggestion
    let t = TempDir::new().unwrap();
```

nitpick, you can `use` this and that's what we usually do